File: app/src/main/java/com/wavesplatform/wallet/ui/dex/details/chart/OnCandleGestureListener.java
package com.wavesplatform.wallet.ui.dex.details.chart;

import android.view.MotionEvent;

import com.github.mikephil.charting.listener.OnChartGestureListener;

public interface OnCandleGestureListener extends OnChartGestureListener {
    void onChartUp(MotionEvent me);
}

File: app/src/main/java/com/wavesplatform/wallet/v2/ui/custom/OnCandleGestureListener.kt
/*
 * Created by Eduard Zaydel on 1/4/2019
 * Copyright © 2019 Waves Platform. All rights reserved.
 */

package com.wavesplatform.wallet.v2.ui.custom

import android.view.MotionEvent

import com.github.mikephil.charting.listener.OnChartGestureListener

interface OnCandleGestureListener : OnChartGestureListener {
    fun onChartUp(me: MotionEvent)
}
